packages/calcite-components/src/components/flow-item/flow-item.stories.ts

```diff
@@ -65,15 +65,17 @@ const contentHTML = html`
 `;
 
 const footerHTML = html`
```

Comment:
Not critical for this PR, but let's try to rename this to better match its usage. This is named as footer, but it has HTML with start/end footer slot content. Applies to panel stories.